example/extensions/lib_subgraph/subgraph_lib.cc

```diff
@@ -177,7 +177,7 @@ REGISTER_OP(_custom_subgraph_op)
 const std::vector<std::string> op_names({"exp","log"});
 
 MXReturnValue mySupportedOps(std::string json,
-                             std::vector<bool> ids,
+                             std::vector<bool>& ids,
```

Comment:
can you elaborate this ids in either the code or the doc? like "this vector flags all the ops in the given symbol graph to be partitioned into subgraphs"

Reply:
In the doc it says
> The 2nd argument is an array of booleans, one for each operator in the model. When traversing the graph, operators to be partitioned into subgraphs are identified and an entry is set to `true` for the index in the `ids` array corresponding to the node ID.

Is that sufficient?